libafl_qemu/Cargo.toml

```diff
@@ -46,6 +46,9 @@ thread_local = "1.1.4"
 capstone = "0.11.0"
 #pyo3 = { version = "0.15", features = ["extension-module"], optional = true }
 pyo3 = { version = "0.17", features = ["pyproto"], optional = true }
+rangemap = "1.0"
+lazy_static = "1.4.0"
```

Comment:
Instead of lazy_static, maybe take a look at https://crates.io/crates/static_init or just a global with unsafe 😬
Iirc there was some runtime overhead otherwise (which probably wouldn't matter for drcov, it's slow anyway, bit still)

Reply:
Drcov generation will only run during instrumentation rather than execution so should not be a hot code path, but makes sense to use a consistent pattern nonetheless.